tensorflow/core/ops/string_ops.cc

```diff
@@ -280,4 +280,32 @@ input: Base64 strings to decode.
 output: Decoded strings.
 )doc");
 
+REGISTER_OP("Substr")
```

Comment:
I was thinking about doing a broadcasted version of this Op, but I wasn't exactly sure how to approach it, so I pushed what I had. I'll give this a go over the next few days!